### ISSUE TITLE
Usage with Jest

### ISSUE THREAD

**Issue by kevgathuku** [smacker/jasmine-expect-jsx#1]
Hello,
I am trying to use `jasmine-expect-jsx` with jest and the example shown below but I am still getting the following error:
`TypeError: expect(...).toEqualJSX is not a function`

```
jest.dontMock('../Main.jsx');

let React = require('react');
require('jasmine-expect-jsx');

describe('Main Component', () => {
  it('renders the Main component', () => {
    expect( < div / > ).toEqualJSX( < div / > );
  });

});
```

What is the recommended way to use this package with jest?

**Comment by smacker** [comment]
I have never used jest, but I created example repo for you: https://github.com/smacker/jest-expect-jsx-example

I'm not sure that it's the most right way to add jasmine plugins to jest, but it works.
if someone knows how to do it simpler - pull requests are welcome.

**Comment by carcer** [comment]
@smacker Latest release breaks with jest :( Have forked and updated the example: https://github.com/carcer/jest-expect-jsx-example

If I get time this weekend I will try and work out what the problem is and raise a PR, however 1.0.4 works, and I am going to use that just now.

**Comment by smacker** [comment]
released 1.1.2. Example works again.

**Comment by smacker** [comment]
Instruction how to use it with Jest was added in README.
